### PR TITLE
Script to find all Environment Variables

### LINKED ISSUE #423: Python Script to find all Environment Variables in a Codebase
# Description

Create a Python Script to find all Environment Variables in a Codebase. It is usual to entirely forget about documenting the Environment Variables in your Project. Your Script should parse the entire codebase and find all the necessary environment variables and create a `env.example` file as the output.

## Language

- [X] Python

## Type of issue

_Please delete options that are not relevant._

- [X] Feature

### PR DESCRIPTION
**Pull request by SmritiAgrawal04**
# Description

The script runs through the code base to fetch all the environment variables used.

Fixes #423

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by SmritiAgrawal04** [comment]
Hi @vybhav72954 @HarshCasper 
Did you get a chance to review this?

File: Python/Environment_Variables/env.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+			# extract env name according to get pattern after spliting the string
+			env= sequence.split('get(\'')[1].split('\')')[0]
+			environmentVariables.add(env)
+
+# open the file to store output
+fout= open('./Environment_Variables.txt', 'w')
+environmentVariables= list(environmentVariables)
+# write all environment variables in the output file
+for env in environmentVariables:
+	fout.write(env)
```
Comment (vybhav72954) [inline Python/Environment_Variables/env.py:42]: Missing an EOF here

File: Python/Environment_Variables/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Find Environment Variables
+
```
Comment (vybhav72954) [inline Python/Environment_Variables/README.md:2]: Can we have a more elaborate README, I think an image or 2 would be a great idea.
Author section too maybe?

File: Python/Environment_Variables/env.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+'''
```
Comment (vybhav72954) [inline Python/Environment_Variables/env.py:1]: We don't need a script docstring (I cant recall the exact name), either put this in README. or get rid of it all together

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: Python/Environment_Variables/env.py (hunk 1 above)
Comment (SmritiAgrawal04) [reply Python/Environment_Variables/env.py:37]: Done @vybhav72954

File: Python/Environment_Variables/env.py (hunk 3 above)
Comment (SmritiAgrawal04) [reply Python/Environment_Variables/env.py]: Done @vybhav72954

File: Python/Environment_Variables/README.md (hunk 2 above)
Comment (SmritiAgrawal04) [reply Python/Environment_Variables/README.md:15]: Elaborated the README. I'm not sure what are you expecting in the author's section.

Comment (vybhav72954) [reply Python/Environment_Variables/README.md:15]: Looks good to me

**Comment by SmritiAgrawal04** [comment]
Can we consider merging the PR?

**Comment by vybhav72954** [comment]
Thanks @SmritiAgrawal04 You are awesome 😄

**Pull request merged**